lang/ilias_tr.lang

```diff
@@ -1343,7 +1343,7 @@ chatroom_adm#:#client_settings#:#Genel Ayarlar
 chatroom_adm#:#enable_osd#:#tarihinde Ekran Bildirimleri Etkinleştir
 chatroom_adm#:#general_settings_title#:#Genel Sohbet Ayarları
 chatroom_adm#:#https#:#HTTPS
-chatroom_adm#:#osd_intervall#:#Refreshınterval (saniye)
+chatroom_adm#:#osd_interval#:#Refreshınterval (saniye)
```

Comment:
Please only touch the `de` and `en` language files. This change has to be reverted.